src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -159,6 +159,13 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             } ~
             path("unlock") {
               passthrough(workspacePath + "/unlock", HttpMethods.PUT)
+            } ~
+            path("storageCostEstimate") {
+              get { requestContext =>
+                perRequest(requestContext,
+                  WorkspaceService.props(workspaceServiceConstructor, userInfo),
+                  WorkspaceService.GetStorageCostEstimate(workspaceNamespace, workspaceName))
+              }
```

Comment:
can we expose a pass through that is the storage in bytes as well? Seems useful from a general api perspective. Or can we include the size in the cost estimate response?